src/main/kotlin/no/nav/familie/baks/mottak/task/OpprettSøknadBehandlingISakTask.kt

```diff
@@ -93,14 +93,10 @@ class OpprettSøknadBehandlingISakTask(
     }
 
     private fun utledBehandlingstype(fagsak: RestMinimalFagsak): BehandlingType {
-        val erFagsakLøpendeOgAktiveBehandlingerAvsluttet =
-            fagsak.behandlinger.any { behandling ->
-                val kanOppretteBehandling = behandling.status == BehandlingStatus.AVSLUTTET && behandling.aktiv
-                fagsak.status != FagsakStatus.LØPENDE && kanOppretteBehandling
-            }
+        val erFagsakLøpende = fagsak.status == FagsakStatus.LØPENDE
 
         val type =
-            if (erFagsakLøpendeOgAktiveBehandlingerAvsluttet || fagsak.behandlinger.isEmpty()) {
+            if (!erFagsakLøpende) {
```

Comment:
Jeg hadde likt å sett en enhetstest på dette, men mulig det er vanskelig å skrive tester for dette uten refaktorering?

Reply:
Det skal være helt mulig 🙌 

Det blir egentlig en test på hele tasken.
Jeg merger denne pr'n også mekker jeg en test på det i separat PR